package.json

```diff
@@ -16,7 +16,7 @@
         "npm": ">=5.4"
     },
     "dependencies": {
-        "elasticio-sailor-nodejs": "2.2.0",
+        "elasticio-sailor-nodejs": "2.2.1",
         "elasticio-node": "0.0.7",
```

Comment:
can you bump version of elasticio-node to 0.0.8? (just in case) 0.0.7 has some conflicts with sailor 2.2.0.